controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1740,6 +1754,32 @@ func saveHostProvisioningSettings(host *metal3api.BareMetalHost, info *reconcile
 	return dirty, nil
 }
 
+func (r *BareMetalHostReconciler) saveHostFirmwareComponents(prov provisioner.Provisioner, info *reconcileInfo, hfc *metal3api.HostFirmwareComponents) (dirty bool, err error) {
+	dirty = false
+	if reflect.DeepEqual(hfc.Status.Updates, hfc.Spec.Updates) {
+		info.log.Info("Not Saving HostFirmwareComponents Information since is not necessary")
```

Comment:
nit: inconsistent case in log messages (I think we use lower case except for CRD's - check other examples)

Reply:
I did that because of  saveHostProvisioningSettings , I will push a change to address.